DiplomaTests/DiplomaTests/Pages/HomePage/HomePage.cs

```diff
@@ -0,0 +1,28 @@
+﻿using DiplomaTests.Navigation;
+using DiplomaTests.Utility;
+using OpenQA.Selenium;
+
+namespace DiplomaTests.Pages.HomePage
+{
+    public class HomePage : PageBase
+    {
+        private WebElementWrapper TitleText => FindElement(By.XPath("//div[@class='oxd-topbar-header-title']"));
+        private WebElementWrapper QuickLaunchTable => FindElement(By.XPath("//div[@class='oxd-grid-3 orangehrm-quick-launch']"));
+
+        public LeftMenuNavigation LMN => new LeftMenuNavigation();
+
+        public HomePage() : base()
+        {
+        }
+
+        public string GetTitleText()
+        {
+            return TitleText.GetText();
+        }
+
+        public void ValidateThatElementsAreDisplayed()
+        {
+            QuickLaunchTable.IsDisplayed();
```

Comment:
IsDisplayed() returns true or false
your method returns void

it means that either element is displayed or not, method will be executed successfully with no errors